library/components/core/WriterTask.bs

```diff
@@ -88,6 +88,14 @@ function getWriteableFile(eventSize as integer) as string
     filenames.Sort("r")
     if (filenames.count() > 0)
         lastFilename = filenames[0]
```

Comment:
In theory we don't need this assignment, only variable declaration, because now instead of taking first element we are going to take first element satisfying the condition.

But I guess it is left like that because in BrightScript it is not possible to declare the variable without assignment?

Reply:
Yes exactly